getnyumba/node-express-api
# #168494708 Setup social Authentication Google/Facebook

#### Description
Setup Authentication using google and facebook
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test

Give instructions on how reviewers can verify your work

- fetch branch using `git fetch ft-social-authentication-168494708 && git checkout ft-social-authentication-168494708 `
- start dev server using `yarn start:dev`
- Obtain an access_token for google from https://developers.google.com/oauthplayground/
- Hit the endpoint using a get method `localhost:8000/api/v1/google?accessToken="yourToken"`
-In the response there will be the user profile

#### Any background context you want to add


[Finishes #168494708]
